src/pcre2_pattern_info.c

```diff
@@ -384,11 +384,10 @@ while (TRUE)
 #endif
     break;
 
-#if defined SUPPORT_UNICODE || PCRE2_CODE_UNIT_WIDTH != 8
```

Comment:
This should have the revert comment as well.

Reply:
I thought that it was OK to just leave the (test-only) code here clean of ifdefs. But I can do if you prefer the ifdefs to come back.